iset.mm

```diff
@@ -105750,6 +105750,20 @@ which defines the set we are taking the supremum of must (a) be true at
     QUSUKUNURAKLZVDVGVHVIUQVLURBKLZUMVLVMGUPABUENZUFUSUKUNPVLVDURACQRTUQUSULUOV
     MVEVGVJVKUQVLVMVNUJUSULUOPVMVEURBDQRTUSVAVCPVDVEGVFURUTVBUGRUH $.
 
+  $( Converse of ~ bezout for the gcd = 1 case, sufficient condition for
```

Comment:
... when the GCD equals 1, ...

Reply:
Sounds good to me; I've pushed up a change to incorporate this wording.